compiler-rt/lib/msan/msan_flags.inc

```diff
@@ -22,6 +22,7 @@ MSAN_FLAG(int, origin_history_size, Origin::kMaxDepth, "")
 MSAN_FLAG(int, origin_history_per_stack_limit, 20000, "")
 MSAN_FLAG(bool, poison_heap_with_zeroes, false, "")
 MSAN_FLAG(bool, poison_stack_with_zeroes, false, "")
+MSAN_FLAG(bool, poison_in_padding, true, "")
```

Comment:
why do we need another flag?

Reply:
Let's not add one if we don't have really strong use-case

Reply:
It controls the new, backwards-incompatible behavior

Reply:
Do you want the padding to be poisoned unconditionally, or is there an interaction with existing flags (e.g., poison_in_malloc)?

Reply:
Just rely on  poison_in_malloc

Reply:
Done